### PR TITLE
Clinical assessment form

### PR DESCRIPTION
**Pull request by semujju**

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/003-HMIS_HIVCare_ARTCard-ClinicalAssessmentPage.xml [hunk 1]
```diff
@@ -0,0 +1,1748 @@
...
+        border-radius: 0;
+        margin-bottom: 0;
+        padding-top: 1px;
+        min-width: 100%;
+        }
+    </style>
+    <!-- jquery loaded for nonessential purposes. -->
+    <script type="text/javascript">
+
+        function enable_disable(field, class_name_prefix, conditions, input_type) {
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/003-HMIS_HIVCare_ARTCard-ClinicalAssessmentPage.xml:77]: @semujju This method should be removed

File: omod/src/main/webapp/resources/htmlforms/003-HMIS_HIVCare_ARTCard-ClinicalAssessmentPage.xml [hunk 2]
```diff
@@ -0,0 +1,1748 @@
...
+        } else {
+        /* remove the fade on the fields */
+        group.find("input").fadeTo(250, 1);
+        group.find("select").fadeTo(250, 1);
+        }
+        }
+        });
+        }
+
+        function enable_disable_district_tb_reg(selected_option) {
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/003-HMIS_HIVCare_ARTCard-ClinicalAssessmentPage.xml:129]: @semujju This function should be removed

File: omod/src/main/webapp/resources/htmlforms/003-HMIS_HIVCare_ARTCard-ClinicalAssessmentPage.xml [hunk 3]
```diff
@@ -0,0 +1,1748 @@
...
+
+        jq("#muac_code").find("select").attr("style", "pointer-events: none;");
+
+        jq("#muac_score").find("input[type$='text']").keyup(function() {
+
+        var age      =  <lookup complexExpression="$patient.age"/>;
+
+        var muacScore = jq(this).val();
+
+        if(age &lt; 5) {
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/003-HMIS_HIVCare_ARTCard-ClinicalAssessmentPage.xml:209]: @semujju Ensure proper indentation for all Javascript

**Comment by solemabrothers** [comment]
@semujju please ensure to resolve all these conversations once they are handled, as of now we don't know whether they where done or not

**Pull request closed without merging**